README.md

```diff
@@ -14,8 +14,7 @@ You can follow the [installation instruction](https://ubuntu.com/server/docs/ins
 ## Step 1: Install `git`
 ```
 sudo apt update
-sudo apt upgrade
-sudo apt install git
+sudo apt install -y git
```

Comment:
I prefer to keep the old version.

Reply:
I'm fine with putting back `sudo apt install git` but I don't think we should be forcing the users to upgrade all their other packages unnecessarily.